plasmapy/mathematics/mathematics.py

```diff
@@ -157,3 +158,65 @@ def plasma_dispersion_func_deriv(zeta):
     Zprime = -2 * (1 + zeta * plasma_dispersion_func(zeta))
 
     return Zprime
+
+
+def Fermi_integral(x, j):
+    r"""Calculate the complete Fermi-Dirac integral.
+    j is the index
+    x is the argument
+
+    Parameters
+    ----------
+    x : Quantity
+        Argument of the Fermi-Dirac integral function.
+
+    j : Quantity
+        Order of the Fermi-Dirac integral function.
```

Comment:
and turn this into "Order and index of..."

Reply:
changed to Order/index